libs/stats/odc/stats/model.py

```diff
@@ -41,17 +42,21 @@ def __init__(self, start: Union[str, datetime],
         DateTimeRange(datetime(2019, 3, 1), '3M')
 
         """
+
+        if isinstance(start, str) and start.startswith("%Y"):
```

Comment:
this does not belong here, constructor should only accept valid time range in ISO format.

Reply:
Removed this block from the construction.